### PR TITLE
Last minute improvements

### PR DESCRIPTION
**Pull request by p-m-jakubowski**
## Description

This PR consists of multiple last minute improvements. Since most of them are very small there was no point in splitting this into multiple separate PRs.

- Changes APY to APR
- Adds "Terms" page
- Updates FAQ
- Removes `gas` parameter from transaction options which fixes wallets gas fee overestimation
- Removes duplicated whitespaces in FAQ text
- Opens footer links in new tabs
- Adds "Terms" and "Privacy" to the footer and updates its styling
- Calls bot actions every 3 minutes, not 2 
- Calls bot immediately after app is loaded
- Removes TS `!` usage in favour of error throwing

### CODE REVIEW

**Comment by aaronmgdr** [comment]
#### 📝 Changed routes:


| [**/connect**](https://staked-celo-web-3j5wilqbf-c-labs.vercel.app/connect) |
|:-:|
|<a href="https://staked-celo-web-3j5wilqbf-c-labs.vercel.app/connect">Screenshot of /connect</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/WpDdBEZTGyepk5q9iq9mpgN-G0m2LjW6hdKTDC2S4bOQn4-_3LPZ7t_B6iLgdLwiKuufyx9lSas-GLrn7wvlMA.png">(view full size)</a> |







Commit d1371dc10c90e1fbb95212f2be1eeb0e9bd0f8bc (https://staked-celo-web-3j5wilqbf-c-labs.vercel.app).

File: src/contexts/account/useWithdrawals.ts [hunk 1]
```diff
@@ -100,6 +102,8 @@ const formatPendingWithdrawals = (values: string[], timestamps: string[]): Pendi
   return pendingWithdrawals.reverse();
 };
 
+const pendingWithdrawalsLoadInterval = 60 * 1000;
```
Comment (m-chrzan) [inline src/contexts/account/useWithdrawals.ts:105]: Is this meant to be on a much faster interval than other actions?

File: src/features/swap/hooks/useUnstaking.ts [hunk 2]
```diff
@@ -15,7 +15,10 @@ export function useUnstaking() {
   const { unstakingRate, celoToUSDRate } = useProtocolContext();
   const [stCeloAmount, setStCeloAmount] = useState<StCelo | null>(null);
 
-  const createTxOptions = useCallback(() => ({ from: address! }), [address]);
+  const createTxOptions = useCallback(() => {
+    if (!address) throw new Error('Cannot create tx options without an address');
```
Comment (soloseng) [inline src/features/swap/hooks/useUnstaking.ts:19]: :heart:

**Review by m-chrzan — APPROVED** [review]
Looks good, just one question.

File: src/pages/faq.tsx [hunk 3]
```diff
@@ -1,33 +1,143 @@
...
+        and CELO when being staked and unstaked via the StakedCelo protocol. Prices on decentralized
+        exchanges are subject to market conditions, market fluctuations, and not per se predictable.
+      </FaqItem>
+      <FaqItem question="What is the difference between StakedCelo and stCELO?">
+        StakedCelo is the name of the protocol and stCELO is the ERC20 token users receive when
+        staking their CELO with the protocol.
+      </FaqItem>
+      <FaqItem question="How is StakedCelo secure?">
+        StakedCelo has been audited and the reports are published{' '}
+        <Link url="https://github.com/celo-org/staked-celo/tree/soloseng/add-audit-report">
```
Comment (soloseng) [inline src/pages/faq.tsx:89]: swithch link to master branch

https://github.com/celo-org/staked-celo/tree/master/audit

File: src/contexts/account/useWithdrawals.ts (hunk 1 above)
Comment (p-m-jakubowski) [reply src/contexts/account/useWithdrawals.ts:105]: Yes. Showing up to date pending withdrawals (e.g. after withdrawal call to bot) is important from the user perspective, so it's better to do this more often.

**Review by soloseng — APPROVED** [review]

File: src/pages/faq.tsx (hunk 3 above)
Comment (p-m-jakubowski) [reply src/pages/faq.tsx]: Done

**Pull request merged**